opal/mca/accelerator/rocm/accelerator_rocm_module.c

```diff
@@ -476,6 +478,11 @@ static int mca_accelerator_rocm_get_device(int *dev_id)
     return OPAL_SUCCESS;
 }
 
+static int mca_accelerator_rocm_get_device_pci_attr(int dev_id, opal_accelerator_pci_attr_t *pci_attr)
+{
+    return OPAL_ERR_NOT_IMPLEMENTED;
```

Comment:
You should implement this, too.  https://rocm-developer-tools.github.io/HIP/group__Driver.html#gafe08d5632ebd24edc26a19da380b2120

Reply:
@edgargabriel Hi could you please take a look at the change and let me know if it makes sense to you. Unfortunately I do not have access to rocm devices for testing, so if possible I very much appreciate if you could do a quick sanity test.🙏